src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -309,37 +267,6 @@ describe('ReactIncrementalScheduling', () => {
     expect(ReactNoop.getChildren()).toEqual([span(5)]);
   });
 
-  it('does not perform animation work after time runs out', () => {
```

Comment:
Maybe? Why didn't we already have a `HighPriority` test then? My impression was that this test existed because of the different semantics for Animation priority versus async/deferred priorities.